pages/windows/nvm.md

```diff
@@ -0,0 +1,21 @@
+# nvm
+
+> Install, uninstall or switch between Node.js versions.
+> Supports version numbers like "0.12" or "v4.2", and labels like "stable", "system", etc.
```

Comment:
`0.12` and `v4.2` are seriously old anad unsupported versions of Node.js. Could we update these slightly at all please?

Reply:
Ah I originally copied that from the original tldr-page of `nvm`. Maybe we should update that too since this Windows version doesn’t support installing very old (io.js) versions.